zookeeper/data.py

```diff
@@ -97,10 +98,12 @@ def train_data(self, batch_size):
             .repeat()
             .map(
                 functools.partial(self.map_fn, training=True),
-                num_parallel_calls=tf.data.experimental.AUTOTUNE,
+                num_parallel_calls=tf.data.experimental.AUTOTUNE
+                if self.autotune
+                else 1 * batch_size,
             )
```

Comment:
is 1* batch_size the correct ration?

Reply:
TF recommends using the number of CPU cores, but in my experience using order batch sizes gives slightly better results.